src/Main.java

```diff
@@ -3,6 +3,8 @@ public class Main {
     public static void main(String[] args) {
         Animal a1 = new Dog();
         a1.makeSound();
+        Animal a2 = new Cat();
```

Comment:
Can you please use a descriptive name for the variable, maybe something like `Animal cat= new Cat()?`